polaris-react/src/components/Badge/Badge.tsx

```diff
@@ -68,68 +51,31 @@ export function Badge({
   const className = classNames(
     styles.Badge,
     status && styles[variationName('status', status)],
-    progress && styles[variationName('progress', progress)],
     icon && styles.icon,
     size && size !== DEFAULT_SIZE && styles[variationName('size', size)],
     withinFilter && styles.withinFilter,
   );
 
-  let progressLabel = '';
-  switch (progress) {
-    case PROGRESS_LABELS.incomplete:
-      progressLabel = i18n.translate(
-        'Polaris.Badge.PROGRESS_LABELS.incomplete',
-      );
-      break;
-    case PROGRESS_LABELS.partiallyComplete:
-      progressLabel = i18n.translate(
-        'Polaris.Badge.PROGRESS_LABELS.partiallyComplete',
-      );
-      break;
-    case PROGRESS_LABELS.complete:
-      progressLabel = i18n.translate('Polaris.Badge.PROGRESS_LABELS.complete');
-      break;
-  }
-
-  let statusLabel = '';
-  switch (status) {
-    case STATUS_LABELS.info:
-      statusLabel = i18n.translate('Polaris.Badge.STATUS_LABELS.info');
-      break;
-    case STATUS_LABELS.success:
-      statusLabel = i18n.translate('Polaris.Badge.STATUS_LABELS.success');
-      break;
-    case STATUS_LABELS.warning:
-      statusLabel = i18n.translate('Polaris.Badge.STATUS_LABELS.warning');
-      break;
-    case STATUS_LABELS.critical:
-      statusLabel = i18n.translate('Polaris.Badge.STATUS_LABELS.critical');
-      break;
-    case STATUS_LABELS.attention:
-      statusLabel = i18n.translate('Polaris.Badge.STATUS_LABELS.attention');
-      break;
-    case STATUS_LABELS.new:
-      statusLabel = i18n.translate('Polaris.Badge.STATUS_LABELS.new');
-      break;
-  }
-
   const accessibilityLabel = statusAndProgressLabelOverride
     ? statusAndProgressLabelOverride
-    : i18n.translate('Polaris.Badge.progressAndStatus', {
-        progressLabel,
-        statusLabel,
-      });
+    : getDefaultAccessibilityLabel(i18n, progress, status);
 
   const hasAccessibilityLabel =
-    progressLabel || statusLabel || statusAndProgressLabelOverride;
+    accessibilityLabel || statusAndProgressLabelOverride;
```

Comment:
we could simplify this down to:
```suggestion
    Boolean(accessibilityLabel)
```

nah?

Reply:
Now that you mentioned, we could actually simplify it down to:
````tsx

let accessibilityMarkup = Boolean(accessibilityLabel)l && (
    <VisuallyHidden>{accessibilityLabel}</VisuallyHidden>
  );
````

Reply:
can't we?